src/DumpCommand.php

```diff
@@ -86,7 +86,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $process = Process::fromShellCommandline($command);
         } else {
             // old Symfony way
-            $process = new Process($command);
+            $process = new Process(explode(' ', $command));
```

Comment:
did process use to take string as arg and now it is taking ...stings or similar?

Reply:
Yes - old Symfony's `Process` method took an array of strings and combined them to make the command.